tests/UnitTests/CoreGraphics.drawing/CGContextDrawingTests.cpp

```diff
@@ -378,3 +378,43 @@ DRAW_TEST(CGContext, PremultipliedAlphaImage) {
     CGContextSetRGBFillColor(context, 1.0, 0.0, 0.0, 0.5);
     CGContextFillRect(context, { 0, 0, 100, 100 });
 }
+
+DRAW_TEST_F(CGContext, AntialiasToggleTranscendsGState, WhiteBackgroundTest<>) {
```

Comment:
I'm not sure that this should be a drawing test, seems more like a regular unit test.